### PR TITLE
Rearranged Text/Hex controls on Find File dialog.

### PR DESCRIPTION
**Pull request by MKadaner**
## Summary
Rearranged Text/Hex controls on Find File dialog.

 
## References
#585

## Checklist
- [X] I have followed the [contributing guidelines](https://github.com/FarGroup/FarManager/blob/master/CONTRIBUTING.md).
- [x] I have discussed this with project maintainers:  <br/>
If not checked, I accept that this work might be rejected in favor of a different great big ineffable plan.<br/>

### CODE REVIEW

File: far/findfile.cpp [hunk 1]
```diff
@@ -729,7 +730,11 @@ intptr_t FindFiles::MainDlgProc(Dialog* Dlg, intptr_t Msg, intptr_t Param1, void
 	{
 		case DN_INITDIALOG:
 		{
-			bool Hex = (Dlg->SendMessage(DM_GETCHECK, FAD_CHECKBOX_HEX, nullptr) == BSTATE_CHECKED);
+			// Set to true if the hotkey of either Text/Hex radio is pressed.
+			// If true, the handler of Text/Hex radio button click will set focus on the Text/Hex editor
+			Dlg->SendMessage(DM_SETITEMDATA, FAD_TEXT_CONTAINING, ToPtr(false));
+
+			bool Hex = (Dlg->SendMessage(DM_GETCHECK, FAD_RADIO_HEX, nullptr) == BSTATE_CHECKED);
```
Comment (alabuzhev) [inline far/findfile.cpp:737]: Can be const.

**Review by alabuzhev — APPROVED** [review]

File: far/findfile.cpp (hunk 1 above)
Comment (MKadaner) [reply far/findfile.cpp]: Sure.

**Comment by MKadaner** [comment]
I checked hotkeys in all languages (and fixed some) and adjusted help articles.
Squashed, rebased, ready.

**Comment by MKadaner** [comment]
Done!

**Comment by alabuzhev** [comment]
@MKadaner currently the radio buttons precede the text/hex input field in the tab order, it might break muscle memory for some ("one tab from the masks field"). What do you think, maybe swap them?

**Comment by MKadaner** [comment]
Hm-mm... An extra tab stop between "Masks" and "Containing." Yeah, it may be annoying. Then the same tab order should be applied to View Search dialog and eventually to the unified search dialog. Let me create a PR.

**Pull request merged**